Dem insertion from file (#1054)
lethe-cfd/lethe: doc/source/parameters/dem/insertion_info.rst

    set list z = 0., 3.

* The ``list velocity x``, ``list velocity y``, ``list velocity z``, ``list omega x``, ``list omega y``, ``list omega z`` and ``list diameters`` define the initial translational velocities, the initial angular velocities and diameters of each particles respectively following the same logic as the insertion coordinates.

---------------------
File
---------------------

> The "file" insertion method insert particles in a similar way to the "list" insertion method. The main difference between these two methods is the need of an external file provided by the ``particles file`` parameter. This parameter is set at ``particles.input`` by default. This external file has to follow this structure:

Reviewer: ```suggestion
The ``file`` insertion method insert particles in a similar way to the ``list`` insertion method. The main difference between these two methods is the option to use an external file provided by the ``insertion file`` parameter. This parameter is set at ``particles.input`` by default. This file has to follow this structure:
```

Reviewer: Also, can you do it for the other methods above (the ``)?

Reviewer: Also with actually the file name and not the file (for the parameter name) but we get it

> The From File insertion method insert particles in a similar way to the List insertion method. The main difference between these two methods is the need of an external file with defined by the ``particles file`` parameter. This parameter is set at ``particles.input`` by default. This external file has to follow this structure:

Reviewer: ```suggestion
The ``from file`` insertion method inserts particles in a similar way to the ``list`` insertion method. The main difference between these two methods is the need of an external file provided by the ``particles file`` parameter. This parameter is set at ``particles.input`` by default. This external file has to follow this structure:
```

.. code-block:: text

    p_x; p_y; p_z; v_x; v_y; v_z; w_x; w_y; w_z; diameters; fem_force_x; fem_force_y; fem_force_z; fem_torque_x; fem_torque_y; fem_torque_z;
    0.0; 0.0; 0.0; 0.0; 0.0; 0.0; 0.0; 0.0; 0.0;       0.2;           0;           0;           0;            0;            0;            0;
    1.0; 2.0; 3.0; 0.0; 0.0; 0.0; 0.0; 0.0; 0.0;       0.2;           0;           0;           0;            0;            0;            0;

> Each line is associated with a particle and gives it its properties. The ``fem_force`` and ``fem_torque`` properties are used in the CFD-DEM solver, but most be specified in all cased. The main advantage of using the ``file`` method over the ``list`` method is that the number of inserted particles in not limited to the number of character accepted on a single line a the parameter file.

Reviewer: ```suggestion
Each line is associated with a particle and its properties. The ``fem_force`` and ``fem_torque`` properties are only used in the CFD-DEM solver, but must be specified in all cases. The main advantage of using the ``file`` method over the ``list`` method is that the number of inserted particles is not limited to the maximum number of characters on a single line of parameter files.
```

Reviewer: Please explain how to generate those files (with the preprocess tool ect..)

Reviewer: As said in private just write something like:
There are 2 options for generating the insertion file. You can either process the data using the code `extract-particles-properties-from-vtu.py` in the` contrib/preprocessing/` directory to pre-process particles from an output file of a previous simulation, or generate the positions and properties of the particles to be inserted using your own code.
